InteractionBase/src/commands/AddReferencedToViewCommand.h

```diff
@@ -96,21 +98,24 @@ CommandResult* AddReferencedToViewCommand<ReferenceTarget, ReferenceResult>::
 
 	auto view = ancestorWithNode->scene()->currentViewItem();
 
-	auto refs = references(DCast<ReferenceTarget>(ancestorWithNode->node()));
+	auto target = DCast<ReferenceTarget>(ancestorWithNode->node());
```

Comment:
Please don't call this `target` as this is the usual name of the second argument of this method and it might just be confusing.